packages/v4/src/content/design-guidelines/charts/themed-chart/themed-chart.md

```diff
@@ -0,0 +1,5 @@
+---
+id: Themed chart
+section: charts
+---
+A **Themed chart** is a chart with different theme and color treatments applied to it. Themed charts can help visually distinguish between different categories and data collection points plotted on the graph.
```

Comment:
This somehow made itself another page - there's one called `Themed chart` and `Themed charts` now

Reply:
I'm not 100% sure we need this like @mcarrano said - it's basically showing you can apply any color theme to a chart and we have set themes (blue family, green family, etc.) to do that with. It might be enough to just link to this from the `Colors for charts page` possibly?

Reply:
Ah! Seems like they made this one plurar! And yeah, we're gonna change that but for now we've decided to just go ahead with these and change it around in a diff PR!